translations/ru/get5.phrases.txt

```diff
@@ -20,6 +20,18 @@
     {
         "ru"            "Напиши {1} когда твоя команда будет готова начать."
     }
+    "YouAreReady"
```

Comment:
This should say "You have been marked as ready", not "READY!".

Similary, `YouAreNotReady `should translate into "You have been marked as NOT ready."